scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -1,12 +1,23 @@
 function populateCytobands(cytobands){
+  var chromPosPattern = new RegExp("^([0-9]{1,2}|[X|T|MT]{1,2})");
```

Comment:
Testing with coordinates I've noticed that the regex allows for searching for chromosomes ranging 23-99, which don't exist of course. For instance if you enter chromosome 99 the form is submitted and the chromosome on top is assigned to "9"